drivers/ma_reports/app/controllers/ma_reports/csg_engage/base_controller.rb

```diff
@@ -0,0 +1,12 @@
+###
+# Copyright 2016 - 2024 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module MaReports::CsgEngage
+  class BaseController < ApplicationController
```

Comment:
This should probably enforce `require_can_view_imports!` to match the menu item.